src/tree/FunctionTreeItem.ts

```diff
@@ -24,7 +24,7 @@ export class FunctionTreeItem extends AzureTreeItem<ISiteTreeRoot> {
     public readonly config: FunctionConfig;
 
     private readonly _name: string;
-    private _triggerUrl: string;
+    private _triggerUrl: string | undefined;
```

Comment:
I'm confused about this change.  Why not keep using `isHttpTrigger` to verify whether or not you can call triggerUrl?

Reply:
I changed `FunctionConfig` to be a lot more permissive since I was using that for local function.json files that the user can edit to a lot of weird things. As part of that, triggerUrl will be undefined if it's not an http trigger instead of throwing an error.

If I do the following, tslint will complain because `writeText` needs `string`, not `string | undefined`
if (node.config.isHttpTrigger) {
        await vscode.env.clipboard.writeText(node.triggerUrl);
}
I _could_ check `isHttpTrigger` _and_ `!node.triggerUrl`, but it was simpler to just check the second one